query_service/core/main.py

```diff
@@ -22,6 +22,7 @@
     "http://127.0.0.1:3001",
     "http://3.134.90.242", #aws ip
     "http://18.119.65.244", #graphdb
```

Comment:
@kabilar @tekrajchhetri -- **note** -- unless you have allocated an AWS Elastic IP address to these IPs, the IPs will change each time if the instance goes up/down/rebooted, etc.

Reply:
@aaronkanzer Thank you for the suggestion. I would work on your suggestion and put it on the active targets.

Reply:
Thanks, Aaron.  That's a good catch.